web/book/src/syntax/expressions-and-operators.md

```diff
@@ -15,21 +15,87 @@ filter circumference > 10 && color != "red"
-or end with an operator require parentheses.
+This table shows operator precedence. Use parentheses `()` to prioritize
+operations and for function calls (see below.) _Question: Are parentheses
+operators? Should they be in this table?_
 
 <!-- markdownlint-disable MD033 — the `|` characters need to be escaped, and surrounded with tags rather than backticks   -->
 
-| Group          | Operators         | Precedence | Associativity |
-| -------------- | ----------------- | ---------- | ------------- |
-| identifier dot | `.`               | 1          |               |
-| unary          | `- + ! ==`        | 2          |               |
-| range          | `..`              | 3          |               |
-| mul            | `* / %`           | 4          | left-to-right |
-| add            | `+ -`             | 5          | left-to-right |
-| compare        | `== != <= >= < >` | 6          | left-to-right |
-| coalesce       | `??`              | 7          | left-to-right |
-| and            | `&&`              | 8          | left-to-right |
-| or             | <code>\|\|</code> | 9          | left-to-right |
-| function call  |                   | 10         |               |
+|          Group | Operators         | Precedence | Associativity |
+| -------------: | ----------------- | :--------: | :-----------: |
+| identifier dot | `.`               |     1      |               |
+|          unary | `- + ! ==`        |     2      |               |
+|          range | `..`              |     3      |               |
+|            mul | `* / %`           |     4      | left-to-right |
+|            add | `+ -`             |     5      | left-to-right |
+|        compare | `== != <= >= < >` |     6      | left-to-right |
+|       coalesce | `??`              |     7      | left-to-right |
+|            and | `&&`              |     8      | left-to-right |
+|             or | <code>\|\|</code> |     9      | left-to-right |
+|  function call |                   |     10     |               |
+|   _parens (?)_ | `()`              |     ??     |   see below   |
+
+## Parentheses
+
+PRQL uses parentheses `()` for several purposes:
+
+- Parentheses group operands to control the order of evaluation, for example:
+  `((1 + x) * y)`
+
+- Parentheses set off an [inner transform](./inner-transforms.md) for the
+  `group ()` and `window ()` transforms. _Question: Are there any other
+  transforms that use parentheses like this?_
+
+- Parentheses set off a minus sign for a function argument, for example:
+  `add (-1) (-3)`
+
+- Parentheses set off nested function calls that contain a pipe, either the `|`
+  symbol or a new line. “Nested” means within a transform; i.e. not just the
+  main pipeline, for example: `(column-name | in 0..20)`
+
+- Parentheses wrap a function call that is part of a larger expression, for
+  example: `round 0 (sum distance)` _Question: Is it necessary to state "on the
```

Comment:
IIUC this is the same point as https://github.com/PRQL/prql/pull/2460/files#r1168000249.

I had this as a footnote so that it was complete. I do think it helps to have the semantics fully listed somewhere, so that we have the true state written down somewhere. But I agree it's not really relevant for users — we could still have it in the footnote, or link to a fuller description in the code / in some tests...

Reply:
see the change that I just committed re: "on the right-hand side..."